packages/cli/src/commands/setup/deploy/__tests__/netlify.test.js

```diff
@@ -1,19 +1,26 @@
 // Automock fs using ../..../__mocks__/fs
-jest.mock('fs')
+vi.mock('fs-extra', async () => {
+  const memfs = await import('memfs')
+  return {
+    default: memfs.fs,
+  }
+})
 
 import path from 'path'
 
 import fs from 'fs-extra'
+import { vol } from 'memfs'
+import { vi, describe, it, expect, beforeEach } from 'vitest'
 
+import '../../../../lib/test'
 import { getPaths } from '../../../../lib'
 import { updateApiURLTask } from '../helpers'
 // Mock telemetry and other things
-import '../../../../lib/test'
 
-jest.mock('../../../../lib', () => {
-  const path = jest.requireActual('path')
+vi.mock('../../../../lib', async (importOriginal) => {
+  // const path = await vi.importActual('path')
```

Comment:
Just adding a comment here so you don't forget about this commented line of code